### ISSUE TITLE
Link between lists and reference section

### ISSUE THREAD

**Issue by rcottrell2310** [JWTC2200/legionbuilder#33]
Great site, it’s become indispensable for me when building and playing. 

Use the reference section a lot in particular. Would be great if you could pull up the reference pages directly from the created list page (I.e. tap on detachment opens a pop up with that detachment stats). 

Would also be great if we could pick loadouts for detachments in the list and the selected options are highlighted in the pop up.

**Comment by JWTC2200** [comment]
First part of this request is done in the form of a popup

**Comment by rcottrell2310** [comment]
Amazing. Thank you!On 21 Dec 2023, at 16:14, Jun Chan ***@***.***> wrote:﻿
First part of this request is done in the form of a popup

**Comment by rcottrell2310** [comment]
I like the new loadouts and appearance but the pop ups from the builder and view screens showing the units stats appear to have gone?

**Comment by JWTC2200** [comment]
They should still be in the builder? 
But you're right I forgot to add it back into the view page.

**Comment by rcottrell2310** [comment]
You’re right they are in the builder. I struggled at first to activate them on my phone as you need quite a precise tap on the ‘?’  Maybe just my old eyes and fat fingers 😜 any chance you could make the landing zone for the tap bigger to make this easier?  Thanks,RichardOn 15 Feb 2024, at 19:22, Jun Chan ***@***.***> wrote:﻿
They should still be in the builder?
But you're right I forgot to add it back into the view page.

**Comment by JWTC2200** [comment]
Ok, fingers crossed, hopefully this is better? XD

**Comment by rcottrell2310** [comment]
That’s so much better! Thanks man. On 15 Feb 2024, at 20:32, Jun Chan ***@***.***> wrote:﻿
Ok, fingers crossed, hopefully this is better? XD
